src/browser/file-recovery-service.js

```diff
@@ -0,0 +1,65 @@
+'use babel'
+
+import {ipcMain} from 'electron'
+import crypto from 'crypto'
+import Path from 'path'
+import fs from 'fs-plus'
+
+export default class FileRecoveryService {
+  constructor (recoveryDirectory) {
+    this.recoveryDirectory = recoveryDirectory
+    this.recoveryPathsByWindowAndFilePath = new WeakMap()
+    this.crashListeners = new WeakSet()
+  }
+
+  start () {
+    ipcMain.on('will-save-path', this.willSavePath.bind(this))
+    ipcMain.on('did-save-path', this.didSavePath.bind(this))
```

Comment:
Yeah, we aren't exercising `start` in our specs and, as you mentioned, the main process will tear everything down when killed, so I figured it wasn't a huge deal to make those handlers disposable. Thoughts? 💭